### ISSUE TITLE
At 400% zoom mode, Content present inside 'Select Data Source' dialog is not properly visible in Windows.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#112087]
### What happened?

**User Impact:**
Low vision user who rely on reflow resolution will be impacted if At 400% zoom mode, Content present inside 'Select Data Source' dialog is not properly visible in Windows.

**Actual Result:**
At 400% zoom mode, Content present inside 'Select Data Source' dialog is not properly visible in Windows.

**Refer Attachment:** 
1. At 400% zoom mode, Content present inside 'Select Data Source' dialog is not properly visible in Windows.mp4
2. At 400% zoom mode, Content present inside 'Select Data Source' dialog is not properly visible in Windows.png

<img width="1107" height="437" alt="Image" src="https://github.com/user-attachments/assets/325d672b-bf86-45d9-96eb-27a70f2d01d4" />
https://github.com/user-attachments/assets/0b5f8882-c25c-453d-9f9a-59d65d6c3586

### What did you expect to happen?

At 400% zoom mode, Content present inside 'Select Data Source' dialog should be properly visible in Windows.

### Did this work before?

No

### How do we reproduce it?

1. Select the download windows AMD64 and install Docker online: [Docker Desktop: The #1 Containerization Tool for Developers | Docker](https://www.docker.com/products/docker-desktop/)
2. Once installed, click "Ok" and wait while the files are unpacked.
3. The Docker screen will appear.
4. Open the command prompt and run as administrator: "docker pull grafana/grafana:12.2.0-17436375126".
5. Next, run this command in the command prompt: "docker run -d --name=grafana12.2_pre -p 3000:3000 grafana/grafana:12.2.0-17436375126".
6. After running the command, go to Images in the left navigation bar to see the container.
7. Click "Run" under the Actions column.
8. Navigate to "Containers" in the left pane and click the link under the "Ports" column in the table.
9. The Grafana link will open; sign in with valid credentials.
10. Go to "Dashboard" in the left navigation bar and click the "New dashboard" button.
11. Click "Add visualization".
12. Press tab key to navigate to the Data Source control. 
13. Press tab key to navigate to the 'Advanced' control and activate it. 'Select Data Source' dialog will appear.
14. At 400% zoom mode, Content present inside 'Select Data Source' dialog is not properly visible in Windows.

### Environment (with versions)?

Grafana: v12.2.0-17436375126 (163ff0c9f1)
OS: Windows 11 Enterprise 24H2 (OS build: 26100.6584)
Browser: Edge (Version 140.0.3485.81), Chrome Version (140.0.7339.186)

### Grafana platform?

Docker

**Comment by joshhunt** [comment]
Thanks for the report. Just to centralise the discussion, I'm going to move this back over to https://github.com/grafana/grafana/issues/106037 👋
